pkg/controllers/manager.go

```diff
@@ -52,9 +51,12 @@ type GenericControllerManager struct {
 func NewManagerOrDie(config *rest.Config, options controllerruntime.Options) Manager {
 	options.Scheme = scheme
 	manager, err := controllerruntime.NewManager(config, options)
-	log.PanicIfError(err, "Failed to create controller manager")
-	log.PanicIfError(manager.GetFieldIndexer().
-		IndexField(context.Background(), &v1.Pod{}, "spec.nodeName", podSchedulingIndex), "Failed to setup pod indexer")
+	if err != nil {
+		panic(fmt.Sprintf("Failed to create controller manager, %s", err.Error()))
```

Comment:
might be cleaner to use `fmt.Fatalf` here?

Reply:
Yeah good point. There are a number of these.

Reply:
Uhhh fmt.Fatalf doesn't exist. I explicitly didn't want to rely on our logging solution in these OrDie startup functions, since in many cases the logger isn't bootstrapped yet.

Reply:
ah oops, `Fatalf` is in the go log pkg.  My bad.